api/storage_proxy.cc

```diff
@@ -143,7 +143,7 @@ void set_storage_proxy(http_context& ctx, routes& r) {
     });
 
     sp::get_rpc_timeout.set(r, [&ctx](const_req req)  {
-        return ctx.db.local().get_config().request_timeout_in_ms()/1000.0;
+        return ctx.db.local().get_config().request_timeout_in_ms().count()/1000.0;
     });
```

Comment:
A nicer way to write it: request_timeout_in_ms() / 1ms.

Reply:
(safer too in case the duration ratio<> changes)

Reply:
ok